cmake/CMakeLists.txt

```
@@ -141,7 +141,7 @@ endif()
 
 # silence nvcc warnings
 if((PKG_KOKKOS) AND (Kokkos_ENABLE_CUDA) AND NOT (CMAKE_CXX_COMPILER_ID STREQUAL "Clang"))
-  set(CMAKE_TUNE_DEFAULT "${CMAKE_TUNE_DEFAULT} -Xcudafe --diag_suppress=unrecognized_pragma -Xcudafe --diag_suppress=128")
+  set(CMAKE_TUNE_DEFAULT "${CMAKE_TUNE_DEFAULT} -Xcudafe --diag_suppress=unrecognized_pragma SHELL:-Xcudafe --diag_suppress=128")
```

I'm surprised that you don't need to group the arguments clearer, like
```suggestion
  set(CMAKE_TUNE_DEFAULT "${CMAKE_TUNE_DEFAULT}" "SHELL:-Xcudafe --diag_suppress=unrecognized_pragma" "SHELL:-Xcudafe --diag_suppress=128")
```

---

@masterleinad thanks for the suggestion. I realized both `-Xcudafe` flags can be combined into one which simplifies everything: `-Xcudafe --diag_suppress=unrecognized_pragma,--diag_suppress=128` and removes the need for the `SHELL:` prefix.